index.html

```diff
@@ -7,11 +7,11 @@
 <body>
     <div class="container">
     <h1>Timer</h1>
-    <button id="startButton">Start</button>
-    <button id="pauseButton">Pause</button>
-    <button id="resumeButton">Resume</button>
-    <button id="stopButton">Stop</button>
-    <p>Timer: <span id="timerDisplay">0</span> seconds</p>
+    <button id="startbutton">Start</button>
```

Comment:
ID should be snake case, follow me: "id="start_button" Do this in the following lines: 10-14.